dags/pull_collisions.py

```diff
@@ -0,0 +1,69 @@
+#!/data/airflow/airflow_venv/bin/python3
+# -*- coding: utf-8 -*-
+# noqa: D415
+r"""### The Daily Collision Replicator DAG
+
+This DAG runs daily to update the following collisions tables from the tables
+staged in the database by the MOVE's ``bigdata_replicator`` DAG:
```

Comment:
Can you please add a note about how these DAGs are triggered by the `bigdata_replicator` DAG? It is not obvious at first when you see the schedule interval.